docs/development/contributing_to_respy.rst

```diff
@@ -0,0 +1,229 @@
+
+Contributions are always welcome. Everything ranging from small extensions of the
+documentation to implementing new features of the model is appreciated. Of course, the
+bigger the change the more it is recommended to reach out to us in advance for a
+discussion. You can post an issue or use the email contact details under
+:ref:`about_us`.
+
+
+Where to start?
+---------------
+
+In general, you can take a look at our `issue tracker <https://github.com/
+OpenSourceEconomics/respy/issues>`_ or at our :ref:`roadmap <roadmap>` to find
+opportunities to contribute to the project.
+
+If you are new to **respy**, you might want to check out issues labeled with
+`good-first-issue <https://github.com/OpenSourceEconomics/respy/issues?q=is%3Aissue+is
+%3Aopen+label%3Agood-first-issue>`_.
+
+
+Workflow
+--------
+
+1. Assuming you have settled on contributing a small fix to the project, fork the
+   `repository <https://github.com/OpenSourceEconomics/respy/>`_. This will create a
+   copy of the repository where you have write access. Your fix will be implemented in
+   your copy. After that, you will start a pull request (PR) which means a proposal to
+   merge your changes into the project.
+
+2. Clone the repository to your disk. Set up the environment of the project with conda
+   and the ``environment.yml``. Implement the fix.
+
+3. We validate contributions in three ways. First, we have a test suite to check the
+   implementation of **respy**. Second, we correct for stylistic errors in code and
+   documentation using linters. Third, we test whether the documentation builds
+   successfully.
+
+   You can run all checks with ``tox`` by running
+
+   .. code-block:: bash
+
+       $ tox
+
+   This will run the complete test suite. To run only a subset of the suite you can use
+   the environments, ``pytest``, ``linting`` and ``sphinx``, with the ``-e`` flag of
+   tox.
+
+   Correct any errors displayed in the terminal.
+
+   To correct stylistic errors, you can also install the linters as a pre-commit with
+
+   .. code-block:: bash
+
+       $ pre-commit install
+
+   Then, all the linters are executed before each commit and the commit is aborted if
+   one of the check fails. You can also manually run the linters with
+
+   .. code-block:: bash
+
+       $ pre-commit run -a
+
+4. If the tests pass, push your changes to your repository. Go to the Github page of
+   your fork. A banner will be displayed asking you whether you would like to create a
+   PR. Follow the link and the instructions of the PR template. Fill out the PR form to
+   inform everyone else on what you are trying to accomplish and how you did it.
+
+   The PR also starts a complete run of the test suite on a continuous integration
+   server. The status of the tests is shown in the PR. Reiterate on your changes until
+   the tests pass on the remote machine.
+
+5. Ask one of the main contributors to review your changes. Include their remarks in
+   your changes.
```

Comment:
Github has [CODEOWNERS](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners) for this. I added this file because with more people it makes sense to distribute the workload.